examples/sinatra_example/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,84 @@
+# Contributor Covenant Code of Conduct
```

Comment:
is this something that one of the dependencies requires? I can't remember any requirement to add this in examples.

Reply:
Its just auto generated by bundler.